gtsfm/configs/scene_optimizer_unit_test_config.yaml

```diff
@@ -19,6 +19,7 @@ SceneOptimizer:
 
     verifier:
       _target_: gtsfm.frontend.verifier.ransac.Ransac
+      use_intrinsics_in_verification: False
```

Comment:
common API, and need to throw error if not possible (e.g. DEGENSAC)

Reply:
done